docs/backup/08-01-velero-turorial.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Taking backup using Velero
+type: Tutorials
+---
+
+This tutorial shows how to perform a partial restore of individual applications running on Kyma using Velero. Follow the guidelines to back up your Kubernetes resources and volumes so that you can restore them on a different cluster.
```

Comment:
```suggestion
This tutorial shows how to use Velero to perform a partial restore of individual applications running on Kyma. Follow the guidelines to back up your Kubernetes resources and volumes so that you can restore them on a different cluster.
```